resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json

```
@@ -134,7 +155,8 @@
 			"m_c doesn't like the way that r_c seems to enjoy {PRONOUN/r_c/poss} influence over the Clan.",
 			"Much to m_c's dismay, r_c has asked {PRONOUN/m_c/object} to do another difficult task.",
 			"m_c thinks that r_c is casting a dark shadow over the whole Clan's reputation.",
-			"m_c is worried that r_c might be sending the Clan down a dark path."
+			"m_c is worried that r_c might be sending the Clan down a dark path.",
+			"m_c has been wondering if StarClan has actually been guiding r_c or the Dark Forest."
```

```suggestion
			"m_c has been wondering if StarClan has actually been guiding r_c or if it's really the Dark Forest's influence."
```
Just making it clearer, the previous version was a bit too easy to misunderstand as "is starclan guiding r_c or is starclan guiding the dark forest"

---

Giotcha, I see that now, thanks for the suggestion